Cryptocompare query failed with "You are over your rate limit, please upgrade your account"

## Problem Definition

This happened in the tests here: https://github.com/rotki/rotki/pull/872/checks?check_run_id=581306979

```
>                   raise RemoteError(error_message)
E                   rotkehlchen.errors.RemoteError: Failed to query cryptocompare for: "https://min-api.cryptocompare.com/data/v2/histohour?fsym=DASH&tsym=EUR&limit=2000&toTs=1434470400". Error: You are over your rate limit please upgrade your account!
```

## Task

Find out what's this new error from cryptocompare and handle accordingly.

Edit: What happened is that all 4 retries during the backing off were used up and then Rotki gave up querying cryptocompare. Perhaps ... increase the number of retries?

--- a/.github/workflows/rotki_ci.yml
+++ b/.github/workflows/rotki_ci.yml
@@ -176,6 +176,12 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/.rotkehlchen-test-dir
+          key: ${{ runner.os }}-testdir-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-testdir-
       - name: Setup SQLCipher
         run: |
           if [ ${{ matrix.os }} == 'ubuntu-latest' ];

--- a/rotkehlchen/externalapis/cryptocompare.py
+++ b/rotkehlchen/externalapis/cryptocompare.py
@@ -12,7 +12,6 @@ from rotkehlchen.assets.asset import Asset
 from rotkehlchen.constants import ZERO
 from rotkehlchen.constants.assets import A_BTC, A_USD
 from rotkehlchen.constants.cryptocompare import KNOWN_TO_MISS_FROM_CRYPTOCOMPARE
-from rotkehlchen.constants.timing import QUERY_RETRY_TIMES
 from rotkehlchen.db.dbhandler import DBHandler
 from rotkehlchen.errors import NoPriceForGivenTimestamp, PriceQueryUnknownFromAsset, RemoteError
 from rotkehlchen.externalapis.interface import ExternalServiceWithApiKey
@@ -36,6 +35,7 @@ T_PairCacheKey = str
 PairCacheKey = NewType('PairCacheKey', T_PairCacheKey)
 
 RATE_LIMIT_MSG = 'You are over your rate limit please upgrade your account!'
+CRYPTOCOMPARE_QUERY_RETRY_TIMES = 10
 
 
 class PriceHistoryEntry(NamedTuple):
@@ -147,7 +147,7 @@ class Cryptocompare(ExternalServiceWithApiKey):
         if api_key:
             querystr += f'&api_key={api_key}'
 
-        tries = QUERY_RETRY_TIMES
+        tries = CRYPTOCOMPARE_QUERY_RETRY_TIMES
         while tries >= 0:
             try:
                 response = self.session.get(querystr)
@@ -173,7 +173,7 @@ class Cryptocompare(ExternalServiceWithApiKey):
                     else:
                         log.debug(
                             f'Got rate limited by cryptocompare and did not manage to get a '
-                            f'request through even after {QUERY_RETRY_TIMES} '
+                            f'request through even after {CRYPTOCOMPARE_QUERY_RETRY_TIMES} '
                             f'incremental backoff retries',
                         )
 

